training_scripts/gsgnn_lp/ml_lp_text.yaml

```diff
@@ -23,7 +23,6 @@ gsf:
     hidden_size: 128
     use_mini_batch_infer: true
   lm:
-    lm_train_nodes: 10
```

Comment:
Why remove lm_train_nodes: 10?

Reply:
This will co-train both BERT and GNN models which is not needed for this test: **Movielens, RGCN layer 1, BERT nodes: movie, user , inference: full-graph, negative_sampler: joint, decoder: DistMult, exclude_training_targets: true, save model**
It also causes slowness in link prediction (exceeds time limit), and this YAML is only used for that test.
James is aware of this issue as well.

Reply:
The motivation of that test is to test co-training. You can set --fanout to 4 and lm_train_nodes to 2 to reduce the cost